### DISCUSSION TITLE
Consider using grouped template for minimal and default options

### DISCUSSION THREAD

**Discussion by theimowski** [SAFE-Stack/SAFE-template#593] [Ideas]
That's what e.g. ASP.NET Core Spa template [does](https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/.template.config/template.json#L8) - you install single nuget package but it contains 3 distinct templates

**Comment by isaacabraham** [comment]
So there would be e.g. `dotnet new safe` and `dotnet new safe-minimal`?

**Comment by theimowski** [comment]
yup if I understand it correctly.
the aforementioned template bundles following three templates:
```
ASP.NET Core with Angular                         angular                  C#                Web/MVC/SPA                  
ASP.NET Core with React.js                        react                    C#                Web/MVC/SPA                  
ASP.NET Core with React.js and Redux              reactredux               C#                Web/MVC/SPA 
```

**Comment by isaacabraham** [comment]
This could be something we do as part of the next SAFE Stack template?

**Comment by Larocceau** [comment]
This is the case now!
